socha-sdk/src/framework/sc/shared/InvalidMoveException.kt

```diff
@@ -0,0 +1,9 @@
+package sc.shared
+
+import sc.api.plugins.IMove
+import kotlin.RuntimeException
+
+/** NOTE: It is important to have information about the move in the reason
+ *  because that is the only place where the invalid move is logged */
+class InvalidMoveException @JvmOverloads constructor(reason: String, @JvmField val move: IMove? = null):
+        RuntimeException(reason + (if (move != null) "; move was $move" else ""))
```

Comment:
```suggestion
        RuntimeException(reason + move?.let { "; move was $move" } ?: ""))
```
please check though, didn't test

Reply:
That was my first implementation; sadly, it appended a literal "null"

Reply:
I think you need parantheses then ;)

Reply:
I'll try